langlib/lang.stream/src/main/ballerina/stream.bal

```diff
@@ -115,6 +182,31 @@ public isolated function reduce(stream<Type,ErrorType?> stm,
 #
 # The parameter `func` is applied to each member of parameter `stm` stream in order.
 #
+# ```ballerina
+# Employee[] employeeList = [
+#     {employeeID: "AA1298", designation: "PM", age: 46, salaryInUsd: 5000},
+#     {employeeID: "BB4562", designation: "Developer", age: 34, salaryInUsd: 3000},
+#     {employeeID: "CC2398", designation: "Manager", age: 45, salaryInUsd: 4500}
+# ];
+#
+# stream<Employee> employeeStream = employeeList.toStream();
+#
+# employeeStream.forEach(function(Employee employee) {
+#     processPayroll(employee.employeeID, employee.salaryInUsd);
```

Comment:
Do we need to call a function in here. Shall we add a simple logic.
```
if employee.employeeID == "AA1298" { 
        employee.salaryInUsd = 5500;
}
```

Reply:
> Do we need to call a function in here. Shall we add a simple logic.
> 
> ```
> if employee.employeeID == "AA1298" { 
>         employee.salaryInUsd = 5500;
> }
> ```

Yes, but I feel `'map` is the correct function above example. Isn't it?

Reply:
```
employeeStream.forEach(function(Employee employee) {
   if employee.employeeID == "AA1298" { 
        employee.salaryInUsd = 5500;
  }

})
```